src/components/MainNav/ProfileDropdown/ProfileDropdown.js

```diff
@@ -89,8 +89,15 @@ class ProfileDropdown extends Component {
 
   createLink(link, index, module) {
     const { stripes } = this.props;
-    const { check } = link;
-    const checkfn = !check ? undefined : (module.getModule()[check] || validations[check]);
+    const { check, route } = link;
+    const checkingFoo = module.getModule()[check] || validations[check];
+    let checkfn;
+
+    if (route === '/settings/myprofile/password') {
```

Comment:
As I understand this dropdown is always visible, why we check this route?

Reply:
Current method creates links, and with this route checking we can define if current link is "Change password" and here we should check if user has necessary permission. If there would be created another link this permission won't be check.